js/lib/explore.min.js

```diff
@@ -117,8 +117,8 @@
 
 	/*!
 	 * Stickyfill -- `position: sticky` polyfill
-	 * v. 1.1.10 | https://github.com/wilddeer/stickyfill
-	 * Copyright Oleg Korsunsky | http://wd.dizaina.net/
+	 * v. 1.1.11 | https://github.com/18F/stickyfill
+	 * Copyright Brian Hedberg | 
```

Comment:
Huh, that's interesting. Let's merge this for now, and follow up with 18F folks on if there's a better method. I know we've talked about how to deal with NPM modules in the past, and I feel like there wasn't a straightforward answer.